### PR TITLE
fvpn: initial stable release v0.2.1

### PR DESCRIPTION
**Pull request by anatolio-deb**
Signed-off-by: Anatolio Nikiforidis anatoly@forestvpn.com

Maintainer: me
Compile tested: main/arm64
Run tested: main/arm64

### CODE REVIEW

File: net/fvpn/Makefile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  DEPENDS:=$(GO_ARCH_DEPENDS) +wireguard-tools
+endef
+
+define Package/fvpn/description
+  ForestVPN - Fast, secure, and modern VPN.
+endef
+
+define Package/fvpn/description
+  $(call Package/fvpn/description)
+endef
```
Comment (BKPepe) [inline net/fvpn/Makefile:48]: Why are you calling the same?

File: net/fvpn/Makefile [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/forestvpn/cli
+GO_PKG_BUILD_PKG:=$(GO_PKG)/main
+GO_PKG_LDFLAGS_X:=$(GO_PKG).appVersion=$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/fvpn/template
```
Comment (BKPepe) [inline net/fvpn/Makefile:30]: No need to use template.

**Review by BKPepe — CHANGES_REQUESTED** [review]
I briefly checked it and found something what could be improved. Anyway, I am not sure if this can be merged when looking into https://forestvpn.com/pricing/ , so I will leave it to the other guys to decide.

File: net/fvpn/Makefile [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+#
+# Copyright (C) ForestVPN.com
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fvpn
+PKG_VERSION:=0.2.1-post.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```
Comment (BKPepe) [inline net/fvpn/Makefile:11]: Oh, this is wrong. The tarball will be downloaded as "v$(PKG_VERSION).tar.gz" in dl folder w/o any name.

**Comment by anatolio-deb** [comment]
> I briefly checked it and found something what could be improved. Anyway, I am not sure if this can be merged when looking into https://forestvpn.com/pricing/ , so I will leave it to the other guys to decide.

Hi, thank you for your review. We are going to change our business plan and make some of our resources available fo free and without ads.

File: net/fvpn/Makefile [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+# Copyright (C) ForestVPN.com
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fvpn
+PKG_VERSION:=0.2.1-post.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/forestvpn/cli/tar.gz/refs/tags/v$(PKG_VERSION)
```
Comment (1715173329) [inline net/fvpn/Makefile:12]:
```suggestion
PKG_SOURCE_URL:=https://codeload.github.com/forestvpn/cli/tar.gz/v$(PKG_VERSION)?
```

File: net/fvpn/Makefile [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/forestvpn/cli
+GO_PKG_BUILD_PKG:=$(GO_PKG)/main
+GO_PKG_LDFLAGS_X:=$(GO_PKG).appVersion=$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+
```
Comment (1715173329) [inline net/fvpn/Makefile:30]:
```suggestion

```

File: net/fvpn/Makefile [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+define Package/fvpn
+  TITLE:=fvpn
+  SECTION:=net
+  CATEGORY:=Network
+  URL:=https://forestvpn.com
+  DEPENDS:=$(GO_ARCH_DEPENDS) +wireguard-tools
+endef
+
+define Package/fvpn/description
+  ForestVPN - Fast, secure, and modern VPN.
```
Comment (1715173329) [inline net/fvpn/Makefile:40]: This description is really tiny, just move it to TITLE.

File: net/fvpn/test.sh [hunk 7]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+case "$1" in
+	"fvpn")
+		fvpn --version 2>&1 | grep "$PKG_VERSION"
```
Comment (1715173329) [inline net/fvpn/test.sh:5]: As there only exists one package, the usage of `case` can be removed.

File: net/fvpn/Makefile (hunk 4 above)
Comment (1715173329) [reply net/fvpn/Makefile]: `?` is necessary here

**Comment by 1715173329** [comment]
Final step is to squash all of your commits into a single one.
But looks like repo managers here do not like this package...

**Comment by anatolio-deb** [comment]
> Final step is to squash all of your commits into a single one.
> But looks like repo managers here do not like this package...

Thank you for your feedback. We are going to provide freeware services in this package.

**Pull request closed without merging**